helpers/package.json

```
@@ -23,7 +23,7 @@
     "node": ">=10"
   },
   "dependencies": {
-    "fast-json-stringify": "^2.4.1"
+    "fast-json-stringify": "^5.0.6"
```

IIUC `^5.0.6` will allow any later 5.1.0, so we'd want something like:

```suggestion
    "fast-json-stringify": "5.0.6"
```

or `"5.0.x"` or something.